setup.py

```diff
@@ -18,6 +18,13 @@
         'httpx >= 0.24.0',
         'PyYAML'
     ],
+    extras_require={
+        "dev": [
+            "pytest",
+            "pytest-asyncio<0.17.0",
```

Comment:
It looks like `pytest.mark.asyncio` has been deprecated at some point ([stackoverflow post](https://stackoverflow.com/a/574631370)) - consequently, I had to pin `pytest-asyncio` to an older version.